lib/watir/elements/image.rb

```diff
@@ -16,18 +16,6 @@ def loaded?
       )
     end
 
-    #
-    # Returns the image's width in pixels.
-    #
-    # @return [Integer] width
-    #
-
-    def width
```

Comment:
But `AttributeHelper` is extended, so element has `.attribute` class method and here you alias an instance method. There shouldn't be a collision.